scripts/neighbor_advertiser

```diff
@@ -243,8 +254,10 @@ def get_vlan_addresses(vlan_interface):
             keys=line.strip().split(' ')
             if keys[0] == 'inet':
                 ipv4_addr.append(keys[1].split('/')[0])
+                ipv4_prefix.append(keys[1].split('/')[1])
```

Comment:
This code assume that all input come in '/' format. Is that the expectation? 260 assumes the same.

Reply:
Line 256 and 259 are obtaining ipv4 and ipv6 addresses respectively based on the same assumption. The sample output also shows the same representation